subscribeButton.js

```diff
@@ -0,0 +1,35 @@
+// Creating a button Element
+const div = document.createElement('div');
```

Comment:
If the subscribe button will always be there (the div), why are we not creating the div in html itself?